src/assets/stylesheets/Instructions.scss

```diff
@@ -59,27 +59,70 @@
     margin-block-start: calc(3 * $space-0-125);
     background-color: $rpf-white;
     mix-blend-mode: difference;
+    min-width: 100%;
```

Comment:
Could you please replace it with logical equivalent? `min-inline-size: 100%`
[min-inline-size](https://developer.mozilla.org/en-US/docs/Web/CSS/min-inline-size)

Reply:
Yeah sure 👍